src/api/registrationApi.ts

```diff
@@ -100,6 +101,57 @@ export const fetchRegistration = async (registrationIdentifier: string, doNotRed
   return fetchRegistrationResponse.data;
 };
 
+interface LogEntryOnBehalfOf {
+  id: string;
+  topLevelOrgCristinId: string;
+  shortName: string;
+  displayName: string;
+}
+
+interface LogEntryPerformedBy {
+  userName: string;
```

Comment:
endre når backend har gjort det
```suggestion
  username: string;
```